api/rhsm-subscriptions-api-spec.yaml

```diff
@@ -1450,6 +1527,106 @@ components:
         cloud_provider:
           description: If applicable, the cloud provider for this host.
           type: string
+    InstanceMeta:
+      title: Root Type for InstanceMeta
+      description: ''
+      type: object
+      properties:
+        count:
+          format: int32
+          type: integer
+        product:
+          $ref: '#/components/schemas/ProductId'
+        service_level:
+          $ref: '#/components/schemas/ServiceLevelType'
+        usage:
+          $ref: '#/components/schemas/UsageType'
+        measurements:
+          type: array
+          items:
+            type: string
+      example:
+        count: 10
+        product: RHEL
+        service_level: Premium
+        usage: Production
+        measurements:
+          - Instance-Hours
+          - Storage-Gigabytes
+          - Transfer-Gigabytes
+    InstanceResponse:
+      title: Root Type for DefaultDataResponse
```

Comment:
Is this title accurate?  Not seeing any other references to `DefaultDataResponse`